### ISSUE TITLE
"libraryDependencies in Test" not taken into account

### ISSUE THREAD

**Issue by backuitist** [mpeltonen/sbt-idea#226]
Currently if you have 2 modules A and B and want a dependency D to be in A compile scope and B test scope you need to to do the following:

Project(A).settings( libraryDependencies += D )
Project(B).settings( libraryDependencies += D.copy( configuration = Some("test") ) )

But the following would be more natural:

Project(A).settings( libraryDependencies += D )
Project(B).settings( libraryDependencies in Test += D )

**Comment by mpeltonen** [comment]
So does the second one work as expected in plain sbt (if we forget sbt-idea)? I've never tried that or even seen that form being used, I think.

**Comment by backuitist** [comment]
Hi, you're right the second form is never used. My mistake! I've tried it in the SBT shell and test:library-dependencies would return a different set of dependencies than compile:library-dependencies, so I expected sbt-idea to add the dependencies scoped in test to the idea test scope.
I wrote something in the SBT google group: https://groups.google.com/forum/?fromgroups#!topic/simple-build-tool/kb439MydLSY
